app/pages/lab/workflow.cjsx

```diff
@@ -105,7 +105,7 @@ EditWorkflowPage = createReactClass
       </h3>
       {if @state.workflowCreationInProgress
         <ModalFormDialog tag="div">
-          <WorkflowCreateForm onSubmit={@props.workflowActions.createWorkflowForProject} onCancel={@hideCreateWorkflow} onSuccess={@handleWorkflowCreation}  projectID={@props.project.id} workflowToClone={@props.workflow} workflowActiveStatus={not @props.project.live} />
+          <WorkflowCreateForm onSubmit={@props.workflowActions.createWorkflowForProject} onCancel={@hideCreateWorkflow} onSuccess={@handleWorkflowCreation}  project={@props.project} workflowToClone={@props.workflow} workflowActiveStatus={not @props.project.live} />
```

Comment:
I wasn't sure if the csjx form of the jsx form is used - happy to remove one of these if it's not needed.

Reply:
Aha - this is still needed. Reading into this more this is the workflow edit form and this links to the copy workflow button which launches the `WorkflowCreateForm` modal. so all this is needed.